src/Navigation.js

```diff
@@ -1,23 +1,24 @@
-import { Link } from "react-router-dom";
+import { NavLink } from "react-router-dom";
+import { isLinkActive } from "./utils";
 
 export default function Navigation() {
   return (
     <nav className="navbar navbar-expand-lg bg-light mb-2">
       <div className="container-fluid">
-        <Link to="/" className="navbar-brand">
+        <NavLink to="/" className="navbar-brand">
           My Blog
-        </Link>
+        </NavLink>
         <div className="collapse navbar-collapse" id="navbarSupportedContent">
           <ul className="navbar-nav me-auto mb-2 mb-lg-0">
             <li className="nav-item">
-              <Link to="/" className="nav-link">
+              <NavLink to="/" className={isLinkActive} end>
```

Comment:
> If the end prop is used, it will ensure this component isn't matched as "active" when its descendant paths are matched.

https://reactrouter.com/en/main/components/nav-link